src/app/tests/suites/TV_AccountLoginCluster.yaml

```diff
@@ -30,11 +30,6 @@ tests:
               - name: "tempAccountIdentifier"
                 value: "asdf"
 
-      # TODO: Enable these once they are able to work
```

Comment:
Why is this being removed?  This seems like a comment for the preceding GetSetupPIN block (and the auto-formatter moved it around weirdly).  This comment should stay.

Reply:
Yeah, moved it back.